barman-montagu

```diff
@@ -82,8 +82,30 @@ def recover(id, wipe_first):
 
 def update_nightly():
     barman = get_barman_container()
-    exec_safely(barman, ["update-nightly", DATABASE_NAME],
+    print("Dumping backup into nightly volume")
+    exec_safely(barman, ["barman", "recover", DATABASE_NAME, "latest",
+                         "/nightly/.incoming"],
                 stream=True, check=True)
+    migrate_nightly()
+    exec_safely(barman, ["update-nightly"], stream=True, check=True)
+
+
+def migrate_nightly():
```

Comment:
'Migrate' being an overloaded term, more clunky 'replay WAL' or similar might be more explicit